tests/integration/security/sds_tls_origination/util/test_certs.go

```diff
@@ -0,0 +1,123 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sdstlsutil
+
+const (
+	ClientCertA = `-----BEGIN CERTIFICATE-----
+MIIDGDCCAgCgAwIBAgIUFW8Q9nJKg/rDzm+EhcqzeULmJs8wDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTQwMDA0NDdaGA8y
+MjkzMTIyODAwMDQ0N1owGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBAL+oQ9HDmyelKSlWkNMSCYl+wrBxb7go
+0iGGalBrxkm0ScWSuGYwjivZfQ1nMvnrZGQZIsw2hDi5b7tgW1QS9G0jAzEPHN/f
+Bunzj3vKRp/U0WflAY8s7mR+bBa9yD9I1JI5kV7ScIt0ICK/dD7LrpT/ikO2xBlS
+hDFWMPV2fGTbneZSX9wDpaGSl4HwwxO9BpZTvE4Lft/gRj8d4eMc6gLETTSAzn24
+MiMy0AUp9r3bX6I/RPrrZDKEhnPG2dW7p7AgOBe7CWa2Zzvi6gnjOc4bqjSLWEgF
+bIGLRdrlYKQn+tzlHjm7+GYoMtGqkRx/pQFAV1SRVcvXjEMOcfHyw7sCAwEAAaNY
+MFYwCQYDVR0TBAIwADALBgNVHQ8EBAMCBeAwHQYDVR0lBBYwFAYIKwYBBQUHAwIG
+CCsGAQUFBwMBMB0GA1UdEQQWMBSCEnNlcnZlci5kZWZhdWx0LnN2YzANBgkqhkiG
+9w0BAQsFAAOCAQEAtztb7XWNXrnq9vYve79kFt6ca8PWK47aZyegh9Elwk6xhcn2
+uCdEIbDdihmNAB66f/QCVPpVJ7C13+bXB2P6DeZ0OPTGyRhjdN6gFPBS3t1qz5vY
+IkN3t5x3RewHB0D1ZPW3QfBg1CtZHohzH+aNtlJLcMuadLxn4uyrsCJPGF37BNiF
+c0Etw3eUBsGqZ88wLLXe5cW536Knz2lLyjo02WueIAaQ/u8f/qRsAyr0uIatYbIT
+5ZtVkfJas3EUb678+oFae0nzwIv20943AAVa5ovPV58lfShnkPTessSH7Q7R1r9b
+0V0pMmlCRCQYVXEjGJk3GUWYYkXbKOFy8fJPqQ==
```

Comment:
What's the expiry date of the cert?  It should be something very large, like 10+ years

Reply:
I'm reusing existing certs from tests/testdata/certs. So expiry should be the same

Reply:
then why not use those certs instead of copying them?  So we only have to update one place when they expire?

Reply:
I was planning on adding some new certs for rotational testing, if I do that I'll make sure the expiry is 10 yrs. If I don't add any other certs, I'll use those directly.